java/core/src/java/org/apache/orc/impl/TreeReaderFactory.java

```diff
@@ -67,7 +70,7 @@
 
     Set<Integer> getColumnFilterIds();
 
-    Consumer<VectorizedRowBatch> getColumnFilterCallback();
+    Consumer<org.apache.orc.filter.FilterContext> getColumnFilterCallback();
```

Comment:
Oh, we cannot import directly like `import org.apache.orc.filter.FilterContext`?

Reply:
This class also uses `org.apache.hadoop.hive.ql.io.filter.FilterContext` that is why this is fully qualified. I am happy to consider a different name e.g. `FilterInput`.

We can make this change after we have an approach on how to handle the breaking change of `Consumer<VectorizedRowBatch>` -> `Consumer<FilterContext>`

Reply:
Will change to OrcFilterContext

Reply:
Merged as part of ORC-755